gen_server.py

```diff
@@ -0,0 +1,38 @@
+import http_server
+
+ADDR = ("127.0.0.1", 8000)
+BUFF = 1024
+
+
+def run_server(conn, adr):
+    request = ""
+    try:
+        while True:
+            s = conn.recv(BUFF)
+            request += s
+            if len(s) < BUFF:
+                break
+        print request
```

Comment:
given that the server process is now going to kick these handlers off in greenlets, I can't help but wonder where the printed output is going to go.  Do you see it in the server terminal when running the server?